content/en/docs/configuration/external/aws-privateca-issuer/_index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "AWS Private CA Issuer"
+linkTitle: "AWS Private CA Issuer"
+weight: 60
+type: "docs"
+---
+
+Kubernetes containers and applications use digital certificates to provide secure authentication and encryption over TLS. A widely adopted solution for TLS certificate life-cycle management in Kubernetes is cert-manager, an add-on to Kubernetes that requests certificates, distributes them to Kubernetes containers, and automates certificate renewal.
+
+ACM Private CA provides an open-source plug-in to cert-manager, [aws-privateca-issuer](https://github.com/cert-manager/aws-privateca-issuer), that allows customers who use cert-manager to improve security over the default cert-manager CA, which stores keys in plaintext in server memory. Customers with regulatory requirements for controlling access to and auditing their CA operations can use this solution to improve auditability and support compliance. You can use the AWS Private CA Issuer plugin with Amazon Elastic Kubernetes Service (Amazon EKS), a self-managed Kubernetes on AWS, or in on-premises Kubernetes.
+
+The diagram below shows some of the options available for using TLS with a Kubernetes cluster. This example cluster, containing various resources, is situated behind a load balancer. The numbers identify possible end-points for TLS-secured communications, including the external load balancer, the ingress controller, an individual pod within a service, and a pair of pods communicating securely with each other.
```

Comment:
Suggestion: (since the diagram and later option descriptions talk about EKS and ELB specifically)
```suggestion
The diagram below shows some of the options available for using TLS in an EKS cluster. This example cluster, containing various resources, is situated behind a load balancer. The numbers identify possible end-points for TLS-secured communications, including the external load balancer, the ingress controller, an individual pod within a service, and a pair of pods communicating securely with each other.
```

Reply:
Applied in AWS tech docs. 

Thanks for the improvements.